packages/loader/container-loader/src/connectionManager.ts

```diff
@@ -1241,9 +1258,11 @@ export class ConnectionManager implements IConnectionManager {
 	};
 
 	// Connection mode is always read on disconnect/error unless the system mode was write.
-	private readonly disconnectHandlerInternal = (disconnectReason: IAnyDriverError): void => {
+	private readonly disconnectHandlerInternal = (disconnectReason?: IAnyDriverError): void => {
```

Comment:
This will always have an error as this handler comes into picture when we get disconnect from connection from service side. From client side when we disconnect, we de register this handler, otherwise it will become a loop. So, we should remove this change in this function.